catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/SceneListFragment.java

```diff
@@ -208,6 +209,8 @@ public void renameItem(Scene item, String name) {
 		if (!item.getName().equals(name)) {
 			if (sceneController.rename(item, name)) {
 				ProjectManager.getInstance().saveProject(getActivity());
```

Comment:
the `ProjectManager.saveProject(..)` starts an async task, so I'm not sure it is the best way to immediately start another task, that works on the same `.xml` file right after that.

Reply:
any suggestions?

Reply:
well, yes and no. I have provided a async task in #3104, with which you could either wait for the task with the listener provided there, or use it sync.